internal/transform/transform.go

```diff
@@ -404,6 +405,17 @@ func ExtractScript(doc *astro.Node, n *astro.Node, opts *TransformOptions, h *ha
 	}
 }
 
+func HintAboutImplicitInlineDirective(n *astro.Node, h *handler.Handler) {
+	if n.Type == astro.ElementNode && n.DataAtom == a.Script && len(n.Attr) > 0 && !HasInlineDirective(n) {
```

Comment:
Script tags only containing `src` also get processed, so we should check for that too
```suggestion
	if n.Type == astro.ElementNode && n.DataAtom == a.Script && len(n.Attr) > 0 && len(n.Attr) == 1 && n.Attr[0].Key != "src" && !HasInlineDirective(n) {
```

Reply:
Oh yeah, that's right!